spec/lib/job_types/campaign_updated_job_spec.rb

```diff
@@ -14,8 +14,8 @@
     parent_campaign_id: parent_campaign.id, slug: 'another-slug-of-slugs-2')}
 
     it 'schedules the child campaigns to update' do
-      expect_job_not_queued.with(JobTypes::ChildCampaignUpdateJob, child_campaign.id)
-      expect_job_not_queued.with(JobTypes::ChildCampaignUpdateJob, child_campaign_2.id)
+      expect_job_queued.with(JobTypes::ChildCampaignUpdateJob, child_campaign.id)
```

Comment:
Why did the test assertion flip 180 degrees?

Reply:
I don't understand why the test ever passed. It shouldn't have. For some reason, once I started working on fixing the deprecations, this spec, correctly, started failing. I wish I had a better answer.